usermanagerlib/src/main/java/org/pesmypetcare/usermanagerlib/datacontainers/Meal.java

```diff
@@ -0,0 +1,65 @@
+package org.pesmypetcare.usermanagerlib.datacontainers;
+
+import androidx.annotation.NonNull;
+import androidx.annotation.Nullable;
+
+public class Meal {
+    private Date date;
+    private MealData body;
+
+    public Meal(Date date, MealData body) {
+        this.date = date;
+        this.body = body;
+    }
+
+    /**
+     * Returns the meal date
+     * @return Meal date
+     */
+    public Date getDate() {
+        return date;
+    }
+
+    /**
+     * Sets a new meal date
+     * @param date Meal date
+     */
+    public void setDate(Date date) {
+        this.date = date;
+    }
+
+    /**
+     * Returns the Meal Data
+     * @return Meal Data
+     */
+    public MealData getBody() {
+        return body;
+    }
+
+    /**
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [First sentence should end with a period.](https://app.codacy.com/gh/Grupo13-PES-Mascotas/PES_My-Pet-Care-Libraries/pullRequest?prid=5314325)